wordpress-mobile/WordPress-iOS
# Remove formattable content parent

Fixes parts of #9758

This PR removes the `FormattableContentParent` protocol, that was created as a temporary helper to facilitate the transition from `NotificationBlock` to `FormattableContent`.

As a result, not just `FormattableContent` became simpler, but also the base implementation `FormattableTextContent`, moving all the Notification specific data to `NotificationTextContent`.

To test:

- Build and run the project.
- Check that Notifications keep working as expected.
- Check that Activity Log keep working as expected.
